backend/services/implementations/testService.ts

```diff
@@ -1,22 +1,47 @@
 import mongoose from "mongoose";
-import MgTest, { AssessmentStatus, Test } from "../../models/test.model";
+import MgTest, {
+  AssessmentStatus,
+  ImageMetadata,
+  QuestionComponent,
+  QuestionComponentType,
+  Test,
+} from "../../models/test.model";
 import {
-  CreateTestRequestDTO,
+  TestRequestDTO,
   TestResponseDTO,
   ITestService,
 } from "../interfaces/testService";
 import { getErrorMessage } from "../../utilities/errorUtils";
 import logger from "../../utilities/logger";
+import IImageStorageService from "../interfaces/imageStorageService";
+import FileStorageService from "./fileStorageService";
+import ImageStorageService from "./imageStorageService";
 
 const Logger = logger(__filename);
 
 class TestService implements ITestService {
+  imageStorageService: IImageStorageService;
+
+  constructor() {
```

Comment:
not sure what is better practice - to initialize `fileStorageService` inside or pass as independent object. but it reduces code and file service shouldn't live outside test service

Reply:
i think it makes sense for each test object to own its own images, but i think if you're not using `fileStorageService` inside `testService`, it probably makes sense to initialize it within `imageStorageService`, unless there's something i'm missing!